intern/service/boss/v1/volume.py

```diff
@@ -143,26 +143,40 @@ def get_cutout(
             session (requests.Session): HTTP session to use for request.
             send_opts (dictionary): Additional arguments to pass to session.send().
             no_cache (optional [boolean]): specifies the use of cache to be True or False.
+            chunk_size (optional Tuple[int, int, int]): The chunk size to request
 
         Returns:
             (numpy.array): A 3D or 4D numpy matrix in ZXY(time) order.
 
         Raises:
             requests.HTTPError
         """
-
-        # Check to see if this volume is larger than 1GB. If so, chunk it into
-        # several smaller bites:
-        if (
+        chunk_size = kwargs.pop("chunk_size", (512, 512, 16 * 8))
```

Comment:
I think `get` would be more appropriate than `pop` so you're not mutating the input params.

Reply:
This was deliberate, since [the later call to `get_request_cutout`](https://github.com/jhuapl-boss/intern/blob/467a7fbafcae58145b4619a303a2578a91fcc3c9/intern/service/boss/v1/volume.py#L207) doesn't expect this new kwarg that I added to the `get_cutout` call; I wanted to avoid modifying other APIs where possible.

Thoughts? Happy to change either way!

Reply:
Ok, sounds like a valid reason until we get sick of the original design of intern and refactor the whole thing. 😄